exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -84,6 +84,29 @@
         document.getElementById(warningElemId).style.display="none";
     }
 
+    //Injects Estimated Rows
+    function injectEstimatedRows() {
+      Object.keys(opRowCountMap).forEach(key => {
+        var tgtElem = $("td.estRowsAnchor[key='"+key+"']"); 
+        var status = tgtElem.append( "<div class='estRows' title='Estimated'>("+opRowCountMap[key]+")</div>" );
```

Comment:
```suggestion
        var status = tgtElem.append("<div class='estRows' title='Estimated'>(" + opRowCountMap[key] + ")</div>");
```

Reply:
Spaces are still present around the expression inside parentheses. This is inconsistent with the overall code style.